Makefile

```diff
@@ -375,7 +384,8 @@ docker-push:
 	if [ $$CONFIRM != "y" ] && [ $$CONFIRM != "Y" ]; then \
 		echo "Exiting." ; exit 1 ; \
 	fi
-	for component in agent cassandra-schema es-index-cleaner es-rollover collector query ingester example-hotrod tracegen; do \
+	DOCKERX_OUTPUT="--push --platform=$(PLATFORMS)" $(MAKE) docker-images-only
```

Comment:
I am very confused about the way` DOCKERX_OUTPUT` is being changed in the Makefile. In the beginning its being set as `DOCKERX_OUTPUT?=--load --platform=linux/amd64` and here we are changing it. What is the idea behind it ? Also I would rename it to something like `DOCKER_BUILDX_PARAMS` because it's more than just `--output` flag.

Reply:
as I said before. buildx will create a builder as backend - [buildkit](https://github.com/moby/buildkit),
which will be running in container.

`docker buildx build` send context to the buildkit and build image layers in container.

`docker buildx build --load` will export the built image layers back to host (for local test in ci)
`docker buildx build --push` will push the built image layers to remote registry like docker.io.

however `docker buildx build --load` only support only one platform `--platform=linux/amd64`

so i have to use those to switch local build or push to remote
`DOCKERX_OUTPUT=--load --platform=linux/amd64` 
`DOCKERX_OUTPUT=--push --platform=linux/amd64,linux/arm64`